src/types.ts

```diff
@@ -117,12 +117,14 @@ interface IGenerateResult {
 
 export interface Response<
   Payload extends unknown = unknown,
-  Event extends boolean = boolean
+  Event extends boolean = boolean,
+  Errors extends unknown = unknown
```

Comment:
We know what type is Errors

```suggestion
  Errors extends Error[] = Error[]
```